### ISSUE TITLE
Tick should return Grade object

### ISSUE THREAD

**Issue by Vichy97** [OpenBeta/openbeta-graphql#425]
Ticks should use a Grade object instead of a string for grade. This should have all fields populated by the server when requesting a tick object.

**Comment by vnugent** [comment]
Indirectly addressed by PR #433

**Comment by Vichy97** [comment]
@vnugent I don't think that PR quite addresses this. It's true, that the climb has a Grade object, but I was planning on allowing the user to specify their proposed grade when ticking (similar to what you can do on MP). I don't think the climb object should be replacement for the existing grade field
